.github/workflows/push-main.yml

```diff
@@ -83,11 +84,11 @@ jobs:
 
       - name: Configuring container registry credentials
         run: |
-          echo "${{ steps.auth.outputs.access_token }}" | skopeo login -u oauth2accesstoken --password-stdin https://europe-docker.pkg.dev
+          echo "${{ steps.auth.outputs.access_token }}" | skopeo login -u oauth2accesstoken --password-stdin https://${{env.IMAGE_REPO}}
 
       - name: 🐳 Sync images to Artifact Registry
         run: |
           skopeo sync \
             --src yaml \
             --dest docker \
-            .github/sync-containers.yml europe-docker.pkg.dev/workspace-images
+            .github/sync-containers.yml ${{env.IMAGE_REPO}}/workspace-base-images/workspace-base-images/workspace-image
```

Comment:
why `workspace-base-images/workspace-base-images/workspace-image` ?

Reply:
Just temporary.  I want to test skopeo accessing GAR, before rebuilding GAR to have a nicer/cleaner URL. This is the URL we used earlier.

CC: @princerachit

Reply:
We'll rebuild GAR and clean-up the URL before merging. 

CC: @princerachit

Reply:
For posterity, I believe the path must be three layers deep. 

When we rebuild GAR, it could look like this:
1. Project -> **gitpod-artifacts**
2. Org -> **docker-dev** (to denote the registry type and general purpose)
3. Repository -> **workspace-base-images** (to denote what this specific repo will contain)

Reply:
@aledbf explaining further what @kylos101 added `workspace-base-images(1)/workspace-base-images(2)/workspace-image(3)`.

(1) Project Name
(2) Repo Name
(3) Image name

Reply:
resolving this comment as we have a [separate issue](https://github.com/gitpod-io/workspace-images/issues/539) to track this